### PR TITLE
Feature/4247 url changed

### PR DESCRIPTION
**Pull request by ivankruchkoff**
## Summary

Addresses issue #4247 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: includes/Core/Authentication/Previous_Connected_Proxy_URL.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	 */
+	const OPTION = Connected_Proxy_URL::OPTION . '_previous';
+
+	/**
+	 * Registers functionality through WordPress hooks.
+	 *
+	 * @since n.e.x.t
+	 */
+	public function register() {
+		add_filter(
```
Comment (eugene-manuilov) [inline includes/Core/Authentication/Previous_Connected_Proxy_URL.php:33]: The `update_option_{name}` is an action, not a filter:
```suggestion
		add_action(
```

**Review by eugene-manuilov — CHANGES_REQUESTED** [review]
Thanks, @ivankruchkoff. This almost looks good. Added two comments, once they are addressed, this PR should be good to go.

File: includes/Core/Authentication/Previous_Connected_Proxy_URL.php (hunk 1 above)
Comment (ivankruchkoff) [reply includes/Core/Authentication/Previous_Connected_Proxy_URL.php]: https://github.com/google/site-kit-wp/pull/4536/commits/1650e8044c8706172b1d75c9952c467b0650eca6

File: assets/js/components/setup/SetupUsingProxy.js [hunk 2]
```diff
@@ -192,10 +200,27 @@ function SetupUsingProxy() {
...
-			'Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.',
-			'google-site-kit'
-		);
+		if ( previousConnectedProxyURL === connectedProxyURL ) {
+			description = __(
+				'Looks like the URL of your site has changed. In order to continue using Site Kit, you’ll need to reconnect, so that your plugin settings are updated with the new URL.',
+				'google-site-kit'
+			);
+		} else {
+			description = createInterpolateElement(
```
[the remark this answers is not in the document]
Comment (ivankruchkoff) [reply assets/js/components/setup/SetupUsingProxy.js]: https://github.com/google/site-kit-wp/pull/4536/commits/f6d6173ef3673c99fa4698716f75ca713ece51a7

**Review by eugene-manuilov — APPROVED** [review]
LGTM. Thanks, @ivankruchkoff.

**Pull request merged**